src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/NetworkInterfaceBasicTest.cs

```diff
@@ -56,7 +56,7 @@ public void BasicTest_AccessInstanceProperties_NoExceptions()
         }
 
         [Fact]
-        [PlatformSpecific(TestPlatforms.Linux|TestPlatforms.Android)]  // Some APIs are not supported on Linux and Android
+        [PlatformSpecific(TestPlatforms.Linux|TestPlatforms.Android)]  // Some APIs are not supported on Linux
```

Comment:
Why change the comment if the test does not run?

Reply:
That's not supposed to be changed. I'll revert that comment change.